book/src/framework/configuration.md

```diff
@@ -1,13 +0,0 @@
-# Configuration
-
-### Environment variables
```

Comment:
This table doesn't fit the viewport and is quite small (I have wide monitor). Please check if you can improve it - I need to scroll a lot to see last column
![Uploading Screenshot 2024-11-08 at 15.19.46.jpeg…]()